frontend/src/pages/search.astro

```diff
@@ -155,7 +158,9 @@ import Header from "../components/Header.astro";
         </div>
       </div>
 
-      {askama.if_("let Some(entity) = entity", <Entity entity="entity" />)}
+      <div class="hidden md:block">
```

Comment:
We should probably move the entity to the top of the searchpage when on mobile, and not completely hide it